service/src/test/java/com/kodemon/service/test/TimeServiceTest.java

```diff
@@ -28,7 +29,8 @@
 
     @Test
     public void currentDateTest() {
-        assertThat(timeService.currentDate(), is(equalTo(new Date())));
+        assertThat(new Date().getTime()-5L, is(lessThan(timeService.currentDate().getTime())));
```

Comment:
nonononono :smile: 
Look, you are basically creating 4 different times here:
1. when you call `new Date().getTime()-5L`
2. when you call `timeService.currentDate().getTime()`
3. when you call `timeService.currentDate().getTime()` 
4. when you call `new Date().getTime()+5L`